doc/sphinxman/document_capabilities.py

```diff
@@ -242,6 +242,7 @@ def extract_modules(winnowed):
     ("cisd", None): (", ci\ *n*", "Arbitrary-order *n* through DETCI is inefficient byproduct of CI", ["fnocc"]),
     ("zapt2", None): (", zapt\ *n*", "Arbitrary-order *n* through DETCI is inefficient byproduct of CI", None),
     ("mp4", None): (", mp\ *n*", "Arbitrary-order *n* through DETCI is inefficient byproduct of CI", ["fnocc"]),
+    ("ccsd(t)", "CCENERGY"): ("FN", "Analytic gradients for conventional all-electron RHF/UHF computations can be requested through |globals__qc_module|\ ``=ccenergy``, but their scaling is best suited to small molecules.", None),
```

Comment:
Memory is what the output files show running out. But if I recall old conversations correctly, it's N^9 scaling, not the expected N^8.

Reply:
I don't think it's N^8 or N^9.  It should be N^7, but the prefactor is terrible in my implementation.

Reply:
eek, yes, N^8 instead of N^7 is what I meant to write. Good to know that the issue is actually prefactor.

If anyone wants to clarify the message, the `document_capabilities.py` file is the single source. Any edits can get propagated to the tables in a future docs build.